### ISSUE TITLE
Update Glottolog entry for Cajun French (Louisiana French).

### ISSUE THREAD

**Issue by gammaandromedae** [glottolog/glottolog#1078]
Currently, Louisiana French is named under the term Cajun French (Glottocode:caju1236) which is something that is currently considered a less accurate term for the family of french dialects spoken in Louisiana. It would be really nice to have its glottolog default name changed to Louisiana French, or in french "Français louisianais" and possibly to have the Glottocode changed to something along the lines of loui####. I have no clue how feasible changing the code is or not but it would be nice to have.

Why this change makes sense:
The Louisiana dialect of French is spoken by a large number of ethnic groups in the state and thus calling it "Cajun" is inaccurate as while the Cajuns are an incredibly important ethnic group for the dialect and the culture, it isn't the only one. Louisiana French's sound and some of its unique vocabulary has been influenced by Old Louisiana Creoles, Cajuns, Haitian Creoles, Americans, Houma, Biloxi, and Spanish. Therefore in academic settings, and more commonly in regular conversation(although the term Cajun French is still used a lot by those outside of the culture), Louisianian french speakers refer to our dialect as Louisiana French (or simply French).

Some examples of the term being used regularly in the Louisianian francophone world:
Louisiana French is used regularly on [Télé-Louisiane](https://telelouisiane.com/) and [Le Louisianais](https://louisianais.com/).
The [Louisiana French Dictionary](https://isbnsearch.org/isbn/1604734035) refers to the dialect as Louisiana French, not Cajun French.

**Comment by d97hah** [comment]
I'll change it to Louisiana French for next version. all the best, H

Pada Rab, 22 Mei 2024 pukul 19.15 Benjamin Eunice ***@***.***>
menulis:

**Comment by xrotwang** [comment]
To be released with Glottolog 5.1 coming week:
```
$ glottolog tree caju1236
INFO    glottolog/glottolog at /home/robert/projects/glottolog/glottolog
Indo-European [indo1319]
   └─ Classical Indo-European [clas1257]
      └─ Italic [ital1284]
         └─ Latino-Faliscan [lati1262]
            └─ Latinic [lati1263]
               └─ Imperial Latin [impe1234]
                  └─ Romance [roma1334]
                     └─ Italo-Western Romance [ital1285]
                        └─ Western Romance [west2813]
                           └─ Shifted Western Romance [shif1234]
                              └─ Northwestern Shifted Romance [nort3208]
                                 └─ Gallo-Rhaetian [gall1280]
                                    └─ Oil [oila1234]
                                       └─ Central Oil [cent2283]
                                          └─ Louisiana French [caju1236]
                                             ├─ Big Woods French [bigw1239]
                                             ├─ Marsh French [mars1252]
                                             └─ Prairie French [prai1242]
```
